nemo/nemo/utils/argparse.py

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2019 NVIDIA Corporation
```

Comment:
Do we need that file at all? Is this part of a bigger plan?

Reply:
It's used in the majority of speech examples.
It serves as a reference for what command line arguments should be so that they can be standardized across all examples. Eg datasets should be passed as --train_dataset and --eval_datasets

Reply:
I know what argparse is. I am asking why you cannot simply "import argparse" in those files. So let me ask once again: "Do we need that file at all? Is this part of a bigger plan?"

Reply:
@tkornuta-nvidia that file contains NemoArgParser with a bunch of params common to nemo apps